src/client/test-run/index.js.mustache

```diff
@@ -41,6 +41,7 @@
         testCafeCore.SETTINGS.set({
             CURRENT_TEST_STEP_NAME:   nextStep ? stepNames[nextStep - 1] : 'Test initialization',
             BROWSER_STATUS_URL:       '{{{browserStatusUrl}}}',
+            TAKE_SCREENSHOTS:         {{{takeScreenshots}}},
```

Comment:
Rename `TAKE_SCREENSHOT_ON_FAILS` to `TAKE_SCREENSHOTS_ON_FAILS` for consistency